GPU/GPUTracking/Merger/GPUTPCGMPropagator.cxx

```diff
@@ -651,6 +651,23 @@ GPUd() int GPUTPCGMPropagator::Update(float posY, float posZ, int iRow, const GP
       rejectChi2 = 1;
     } else {
       int retVal = InterpolateReject(posY, posZ, clusterState, rejectChi2, inter, err2Y, err2Z);
+
+      if (o2::utils::DebugStreamer::checkStream(o2::utils::StreamFlags::streamRejectCluster)) {
+        mStreamer.setStreamer("debug_InterpolateReject", "UPDATE");
+        mStreamer.getStreamer() << mStreamer.getUniqueTreeName("tree_InterpolateReject").data()
```

Comment:
I see several times a setStreamer and then getStreamer call. Can we unify this in a single function taking all arguments. Would reduce one line of code. Sorry for being a bit picky. I really think the functionality is OK, but I am just trying to write it in an as short as possible form, to keep the tracking code as compact as possible. In any case it is pretty long already ... :(